docs/content/documentation/modules/tensor_mechanics/VolumetricLocking.md

```diff
@@ -0,0 +1,50 @@
+#Volumetric locking correction
+
+Volumetric locking is the over stiffening of elements when the material is close to being incompressible (Poisson's ratio $\nu$ nearing 0.5). This happens when a fully integrated element (such as Hex8 elements with 8 quadrature points or Quad4 elements with 4 quadrature points) is used. This is a numerical artifact introduced because shape functions used in finite element analysis cannot properly approximate the incompressibility condition throughout the element. To avoid this locking of elements, B-bar correction [see \citet{hughes1987finite}] is implemented in MOOSE.
+
+In this method, both the strain ($\epsilon$) and the virtual strain ($\delta \epsilon$) in an element are separated into volumetric and deviatoric components. The volumetric component is then replaced with an element averaged volumetric strain. This ensures that the volumetric strain remains constant throughout the element.
+
+For example, in the case of small strain linear elasticity, the equation of motion is:
+
+\begin{align*}
+\int_V \sigma(\epsilon)\delta \epsilon dV - \int_V b \delta v dV - \int_{\partial V} t \delta v dA = 0
+\end{align*}
+
+The element averaged volumetric strain (assuming small strain formulation) is:
+
+\begin{align*}
+ w=\frac{1}{V_e} \int_{V_e} \frac{1}{3} tr(\epsilon) dV,
+\end{align*}
+
+where $V_e$ is the volume of the element and tr(.) is the trace of the matrix.
+
+The strain in each element is replaced by the approximation:
+
+\begin{align*}
+\bar{\epsilon} = \epsilon +(w - \frac{tr(\epsilon)}{3})I,
+\end{align*}
+
+where $I$ is the $3 \times 3$ identity matrix. Similarly, the virtual strain is also approximated by:
+
+\begin{align*}
+\bar{\delta \epsilon} = \delta \epsilon + (\delta w - \frac{tr(\delta \epsilon)}{3})I
+\end{align*}
+
+The modified equation of motion is:
+\begin{align*}
+\int_V \sigma(\bar{\epsilon})\bar{\delta \epsilon} dV - \int_V b \delta v dV - \int_{\partial V} t \delta v dA = 0
+\end{align*}
+
+More details about this method can be found in [Bower(2012)](http://solidmechanics.org/Text/Chapter8_6/Chapter8_6.php#Sect8_6_1).
```

Comment:
can we use bib for that as well? It will make generating nice PDFs much easier going forward